tools/xabuild/XABuild.cs

```diff
@@ -146,6 +146,13 @@ static void CreateConfig (XABuildPaths paths)
 			}
 
 			Directory.CreateDirectory (paths.MSBuildTempPath);
+			// Copy NuGet.Frameworks.dll from output directory to MSBuildTempPath to appease:
+			// https://github.com/dotnet/msbuild/blob/755d4d1e3d2a89f72f659fc3d7d2933cab619828/src/Build/Utilities/NuGetFrameworkWrapper.cs#L32
+			var nuGetFrameworksPath = Path.Combine (AppContext.BaseDirectory, "NuGet.Frameworks.dll");
+			if (!File.Exists (nuGetFrameworksPath))
+				throw new Exception ($"Could not find NuGet assembly {nuGetFrameworksPath} required by MSBuild.");
+
+			File.Copy (nuGetFrameworksPath, Path.Combine (paths.MSBuildTempPath, Path.GetFileName (nuGetFrameworksPath)));
```

Comment:
Yeah, I don't see another way to workaround this...

Instead of `throw` can this `Console.WriteLine()` the error and make the exit code 1?